subprojects/tooling-api/tooling-api.gradle

```diff
@@ -62,6 +63,31 @@ task shadedJar(type: ShadedJar) {
     shadowPackage = "org.gradle.internal.impldep"
 }
 
+task toolingApiShadedJar(type: Zip) {
+    dependsOn shadedJarWithoutVersion, project(':versionInfo').createBuildReceipt
```

Comment:
Why do you need to depend on `createBuildReceipt` when you use `createTestBuildReceipt` below?